doc/source/examples/pdp_regression_bike.ipynb

```diff
@@ -0,0 +1,645 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    !wget https://raw.githubusercontent.com/christophM/interpretable-ml-book/master/data/bike.csv -q
</pre><p>Mmm tricky, since there is no <code>wget</code> on Windows (and Mac?) would need to exclude this notebook from tests.</p><p> </p><p>If we keep <code>wget</code> then should remove the file at the end of the notebook.</p><p> </p><p>Perhaps also worth implementing <code>fetch_bike</code>  but that would add some unnecessary complexity as currently most of our <code>fetch*</code> functions return processed data (e.g. see <code>fetch_adult</code>) ready for consumption of models.     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I would add the <code>fetch_bike</code> in another PR and update the notebook. </p>

Reply:
Resolving as the notebook now reads the csv file directly using `pandas`.